### DISCUSSION TITLE
Kibana dashboards missing after upgrade to 2.3.180

### DISCUSSION THREAD

**Discussion by tfalkowski-nsidc** [Security-Onion-Solutions/securityonion#8945] [Unsupported Versions] [answered]
After upgrading to 2.3.180 kibana is unable to locate the default SO dashboards. When I login to Kibana I am greeted with a blank page telling me to create my first dashboard and an error in a popup in the lower right hand corner of the window with the following message:

Error encountered while loading saved dashboard: Could not locate that dashboard (id: a8411b30-6d03-11ea-b301-3d6c35840645)

![Screen Shot 2022-10-18 at 4 16 55 PM](https://user-images.githubusercontent.com/23199930/196555743-26dceb61-14a6-4985-bd4a-eb97eea2a8ad.png)

**Comment by dougburks** [comment, marked as answer]
Please see:
https://docs.securityonion.net/en/2.3/soup.html#kibana
https://docs.securityonion.net/en/2.3/kibana.html#dashboards

**Reply by tfalkowski-nsidc** [reply]
Hey Doug - 

Thank you for taking the time to help me out, it is appreciated.

As you suggested, the solution to my issue was to run the commands suggested in the docs under [https://docs.securityonion.net/en/2.3/soup.html#kibana](url)

```
sudo rm /opt/so/state/kibana_*.txt
sudo salt-call state.apply kibana.so_savedobjects_defaults -l info queue=True
```

The dashboards were successfully restored.

Thanks again for the help.
